src/Datadog.Trace.ClrProfiler.Native/Resource.rc

```diff
@@ -68,12 +68,12 @@ BEGIN
         BEGIN
             VALUE "CompanyName", "Datadog, Inc."
             VALUE "FileDescription", "Datadog CLR Profiler"
-            VALUE "FileVersion", "1.18.1.0"
+            VALUE "FileVersion", "1.18.2.0"
             VALUE "InternalName", "Datadog.Trace.ClrProfiler.Native.DLL"
             VALUE "LegalCopyright", "Copyright (C) 2017-2019"
```

Comment:
Unrelated, but this should say `Copyright 2018 Datadog, Inc.` to match [NOTICE](https://github.com/DataDog/dd-trace-dotnet/blob/master/NOTICE).

Reply:
We have `<Copyright>Datadog, Inc. 2017-2020</Copyright>` in src/Directory.Build.props. Should we consolidate to this one across the three files, or `Copyright 2018 Datadog, Inc` as you're suggesting?

Reply:
Oh, I didn't mean for you to spend time on this. The `2019` just caught my eye. But since we're here 😄, the [Apache license](https://github.com/DataDog/dd-trace-dotnet/blob/master/LICENSE) says all the way at the bottom that we should use
```
Copyright [yyyy] [name of copyright owner]
```

We should ask the open source team, but I'm guessing we should probably change them all to match that.

Reply:
This one in `Resource.rc` doesn't mention the company name, and the one you pointed out in `Directory.Build.props` doesn't have the word "Copyright" (which I think required by copyright law in the USA, but it's © elsewhere).